src/components/PfSpinner.tsx

```diff
@@ -22,7 +23,7 @@ export const PfSpinner: React.SFC<PfSpinnerProps> = props => {
   const { isLoading } = props;
   // It is more than likely it won't have any children; but it could.
   // @todo: Patternfly Spinner is not working here
-  return <Spinner className={spinnerStyle} loading={isLoading} inverse={true} />;
+  return isLoading ? <SpinnerIcon className={spinnerStyle} /> : <></>;
```

Comment:
Nitpicking: dummy question, should be better to return 'undefined' instead of '<></>'?

Reply:
mmm yep why not xD

Reply:
hahaha not xD 
it should be a ReactElement
```ts
Type 'undefined' is not assignable to type 'ReactElement<any, string | ((props: any) => ReactElement<any, string | ... | (new (props: any) => Component<any, any, any>)> | null) | (new (props: any) => Component<any, any, any>)> | null'. 
```